### ISSUE TITLE
Can't display every changeset

### ISSUE THREAD

**Issue by naoliv** [osmlab/changeset-map#91]
For example, it's not possible to see the changes here https://osmlab.github.io/changeset-map/#45229331

Maybe caused by the big bbox.

Could this be handled somehow?

**Comment by ajithranka** [comment]
@naoliv You are right, the overpass adiff query fails because of the large bounding box.

The response we get back is:
```
{
  "version": 0.6,
  "generator": "Overpass API",
  "osm3s": {
    "timestamp_osm_base": "2017-01-31T10:28:02Z",
    "copyright": "The data included in this document is from www.openstreetmap.org. The data is made available under ODbL."
  },
  "elements": [
  ],
"remark": "runtime error: Query run out of memory using about 2048 MB of RAM."
}
```

I believe it's the same case with achavi, which only shows the bounding box: https://overpass-api.de/achavi/?changeset=45229331.

**Comment by nrenner** [comment]
See also:
- nrenner/achavi#9: Achavi "seems not to work" for some changesets
- drolbr/Overpass-API#322: Optimize runtime and memory footprint of augmented diff queries

**Comment by batpad** [comment]
This should be better for newer changesets as we try and cache the adiff results on s3, but am afraid we may not have a solution for older changesets.
